synapse/storage/schema/__init__.py

```diff
@@ -62,6 +62,9 @@
 Changes in SCHEMA_VERSION = 69:
     - We now write to `device_lists_changes_in_room` table.
     - Use sequence to generate future `application_services_txns.txn_id`s
+
+Changes in SCHEMA_VERSION = 70:
+    - event_reference_hashes is no longer written.
```

Comment:
Would it make more sense to say
```suggestion
    - event_reference_hashes is no longer written to.
```

Reply:
probably! thanks.